### DISCUSSION TITLE
Sandboxing

### DISCUSSION THREAD

**Discussion by AMWMusic** [apohl79/audiogridder#556] [Help]
Audio Gridder 1.1.1 works fine for me, even heavy processing is possible via the network. Trying 1.2.0 with the same projects, that run fine with 1.1.1, results in serious crackles. I assume, that this is due to sandboxing, which is new to 1.2.0. The reason, why I think that, is, that issues of this kind can also occur, if I run the DAW Reaper with dedicated processes for plugins.
Does anyone else experience such issues?
Were it possible to introduce an option to turn on and off sandboxing?

**Comment by jaacMusic** [comment]
I think you mean separate threads. There is an option to disable AG instances to be run in separate threads and have it all run in one single thread. Just note, if this one crashes, it will crash the whole stack.

**Reply by AMWMusic** [reply]
Thank you very much for your comment! I will try that.

**Comment by CaBuffalo** [comment]
You can disable the sandbox in the server setting menu.

Sandbox improves the performance quite a lot. Wondering what your set up is.

On Sat, May 15, 2021 at 6:14 PM jaacMusic ***@***.***> wrote:

> I think you mean separate threads. There is an option to disable AG
> instances to be run in separate threads and have it all run in one single
> thread. Just note, if this one crashes, it will crash the whole stack.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/556#discussioncomment-743227>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6DLHEPJYCFAO43X7ALTN4MAPANCNFSM446LDE4Q>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by AMWMusic** [reply]
Thank you very much for your comment! I will try that.
The set up is three computers, where the main computer is a 6 core Phenom II, whereas the remote computers are an 8 core FX and a 16 core Ryzen. I use Reaper as DAW. When I activate dedicated plugin processes or speparate processes in Reaper, soon crackles occur. And it is the same with Audio Gridder. While I can improve processing with 1.1.1, projects with the new version don't work anymore in the intended way. So I hope, that changing the settings in AG will help.

**Reply by apohl79** [reply]
Check the log files and watch out for buffering warnings. If you get audio dropouts etc you might have to increase the buffer size in AG. You can also play with the buffer settings in your DAW.

**Reply by AMWMusic** [reply]
Thank you very much for your comment! I will try to do that. Normally I run audio project with a 2048 buffer size in my audio interface. When AG has 43ms at the same time, things are normally working and latency is not stongly recognizable. I have to scrutinize these dropouts when having installed the new version once again.
